app/(admin)/admin/notifications/actions/sendEmailNotification.test.tsx

```diff
@@ -0,0 +1,35 @@
+'use server';
+import { messaging } from "@/api/serverConfig";
+import { sendEmailNotifications } from "./sendEmailNotification";
+
+jest.mock('./sendEmailNotification', () => ({
+    sendEmailNotifications: jest.fn(),
+}))
+
+jest.mock('@/api/serverConfig', () => ({
+    messaging: {
+        createEmail: jest.fn(),
+    },
+}));
+
+describe('SendEmailNotification', () => {
+    it('should send email with provided information', async () => {
+        const content = 'Test';
```

Comment:
I wonder if we can make these test values more specific to our application?

Reply:
What do you mean by this? The content and subject is something the admin can customize themselves on the frontend. The value can be any thing.